fe-games/src/components/Review.jsx

```diff
@@ -37,21 +60,29 @@ export function Review() {
         <p>Designed by {currentReview.designer}</p>
         <p>{currentReview.owner}</p>
         <p>{currentReview.review_body}</p>
-        <p>Votes: {currentReview.votes}</p>
+        {err ? <p>{err}</p> : null}
+        {voted ? <p>Removed vote!</p> : <p>Added vote!</p>}
```

Comment:
Nice work here!! 🥳 I'd consider adding separate buttons for upvote/downvote to make it easier for the user to see which button they are pressing. You may want to refactor your functionality to stop the user from voting multiple times, rather than subtracting the vote when they press the same button. Please send an nchelp if you are unsure 👍